internal/pkg/trustprovider/trustprovider.go

```diff
@@ -36,27 +36,18 @@ func (tp *TrustProvider) GetValues() error {
 	switch tp.Kind {
 	case "kubernetes":
 		tp.AgentConfig = TrustProviderAgentConfig{
-			WorkloadAttestor:        KubernetesTrustProvider,
-			WorkloadAttestorEnabled: true,
+			WorkloadAttestor: KubernetesTrustProvider,
 			WorkloadAttestorConfig: map[string]any{
-				"enabled":                     true,
-				"skipKubeletVerification":     true,
-				"disableContainerSelectors":   true,
-				"useNewContainerLocator":      false,
```

Comment:
This defaults to `true` if omitted in the latest version (v0.24.1) of the Helm charts. Being explicit with the values here could be beneficial when we upgrade as part of this [issue](https://github.com/cofide/cofidectl/issues/41)?